src/System.Reflection.Metadata/tests/Metadata/TypeSystem/AssemblyReferenceTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Reflection.Metadata.Tests
+{
+    public class AssemblyReferenceTests
+    {
+        [Fact]
+        public void DefaultAssemblyNameFlagIsNone()
+        {
+            AssemblyName assemblyName = new AssemblyName();
+
```

Comment:
This is just testing the AssemblyName default constructor which doesn't live in this assembly. Was this just to confirm its behavior? We shouldn't leave this out of place test here.